packages/ckeditor5-dev-webpack-plugin/package.json

```
@@ -8,15 +8,16 @@
     "@ckeditor/ckeditor5-dev-utils": "^24.2.0",
     "chalk": "^4.0.0",
     "rimraf": "^3.0.2",
-    "webpack-sources": "^1.4.3"
+    "semver": "^7.3.4",
+    "webpack-sources": "^2.0.1"
   },
   "devDependencies": {
     "chai": "^4.2.0",
     "proxyquire": "^2.1.3",
     "sinon": "^9.0.2"
   },
   "peerDependencies": {
-    "webpack": "^4.43.0"
+    "webpack": "^4.43.0 || ^5.0.0"
```

We tested the changes using the following versions:

```suggestion
    "webpack": "^4.43.0 || ^5.24.0"
```